src/components/PrizePage/Prizes.js

```diff
@@ -40,6 +125,41 @@ export default function Prizes() {
 		prizeImageMap.set(id, childImageSharp.fixed);
 	}
 	return (
-		<ComingSoon alignment='center'/>
+		<>
+			<Container maxWidth='md'>
+				<Grid container spacing={3} className={classes.prizeGrid}>
```

Comment:
is there any way to add some spacing at the bottom of the page at the very end of the prize grid? It looks a bit compact as of now.